FBTweak/FBTweakCategory.m

```diff
@@ -34,40 +37,49 @@ - (instancetype)initWithName:(NSString *)name
 {
   if ((self = [super init])) {
     _name = [name copy];
-    
-    _orderedCollections = [[NSMutableArray alloc] initWithCapacity:4];
-    _namedCollections = [[NSMutableDictionary alloc] initWithCapacity:4];
+    _tweakCollections = [NSMutableArray array];
   }
   
   return self;
 }
 
 - (void)encodeWithCoder:(NSCoder *)coder
 {
-  [coder encodeObject:_name forKey:@"name"];
-  [coder encodeObject:_orderedCollections forKey:@"collections"];
+  [coder encodeObject:self.name forKey:@"name"];
+  [coder encodeObject:self.tweakCollections forKey:@"collections"];
 }
 
 - (FBTweakCollection *)tweakCollectionWithName:(NSString *)name
 {
-  return _namedCollections[name];
-}
-
-- (NSArray *)tweakCollections
-{
-  return [_orderedCollections copy];
+  NSUInteger index = [_tweakCollections indexOfObjectPassingTest:
+                      ^BOOL(FBTweakCollection * _Nonnull collection, NSUInteger __unused idx, BOOL * _Nonnull stop) {
```

Comment:
`^BOOL(FBTweakCollection * _Nonnull collection, NSUInteger __unused idx, BOOL * _Nonnull stop) {`
->
`^BOOL(FBTweakCollection *collection, NSUInteger, BOOL *) {`

Reply:
So what we decided about the indentation of changes you do? Than you use the default Xcode's indentation?

Reply:
I can't do that, it's cpp, and this file is `.m`.